lib/idx/remediators/EnrollPoll.ts

```diff
@@ -0,0 +1,50 @@
+/*!
+ * Copyright (c) 2015-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * 
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+
+import { Remediator, RemediationValues } from './Base/Remediator';
+import { NextStep } from '../../types';
+import { IdxContext } from '../types/idx-js';
+
+
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
+export interface EnrollPollValues extends RemediationValues {
+  startPolling?: unknown;
+}
+
+export class EnrollPoll extends Remediator {
+  static remediationName = 'enroll-poll';
+
+  values: EnrollPollValues;
+
+  canRemediate() {
+    return Boolean(this.values.startPolling);
+  }
+
+  getNextStep(context?: IdxContext): NextStep & {pollInterval: number} {
+    const name = this.getName();
+    let authenticator = this.getAuthenticator();
+    if (!authenticator && context?.currentAuthenticator) {
+      authenticator = context.currentAuthenticator.value;
+    }
+    return { 
+      name, 
+      authenticator,
+      pollInterval: this.remediation?.refresh,
```

Comment:
When in auth-js does this `pollInterval` actually get processed? I would imagine auth-js would encapsulate this polling behavior.

Reply:
It is expected to be processed by the consuming code - when remediation runs on server side, client(rendered page) does not know when to poll.